test/e2e/network/ingress.go

```diff
@@ -831,10 +831,10 @@ func executeStaticIPHttpsOnlyTest(f *framework.Framework, jig *ingress.TestJig,
 
 	ginkgo.By("waiting for Ingress to come up with ip: " + ip)
 	httpClient := ingress.BuildInsecureClient(ingress.IngressReqTimeout)
-	framework.ExpectNoError(framework.PollURL(fmt.Sprintf("https://%s/", ip), "", e2eservice.LoadBalancerPollTimeout, jig.PollInterval, httpClient, false))
+	framework.ExpectNoError(ingress.PollURL(fmt.Sprintf("https://%s/", ip), "", e2eservice.LoadBalancerPollTimeout, jig.PollInterval, httpClient, false))
```

Comment:
seems that these are GCE only tests that doesn't support ipv6, but we can use this opportunity to make it ipv6 ready, what do you think?
I don't know if there is something easier than `net.JoinHostPort(ip,"443)` per example

Reply:
I'd like to avoid mix multiple purposes into a single PR for long-term maintenance.
That means sometimes we want to revert some PRs which make some wrong thing.
If mixing multiple purposes, everything in the PR is reverted.

Reply:
absolutely